PowerUp/app/src/main/res/layout/pgame.xml

```diff
@@ -21,16 +21,38 @@
         android:textColor="@android:color/black"
         android:textSize="@dimen/game_text_size" />
 
-    <ListView
-        android:id="@+id/mainListView"
+    <ImageView
+        android:id="@+id/askerImageView"
+        android:layout_width="@dimen/asker_image_view_width"
+        android:layout_height="@dimen/asker_image_view_height"
+        android:layout_alignParentRight="true"
+        android:layout_below="@id/scenarioNameEditText"
+        android:src="@drawable/marco" />
+
+
+    <LinearLayout
         android:layout_width="@dimen/main_list_view_width"
         android:layout_height="@dimen/main_list_view_height"
         android:layout_alignParentRight="true"
-        android:layout_below="@+id/askerImageView"
-        android:background="@android:color/white"
-        android:backgroundTint="@color/abc_secondary_text_material_dark"
-        android:clickable="true" >
-    </ListView>
+        android:layout_below="@id/askerImageView"
+        android:background="@android:color/black">
+        <ListView
+            android:id="@+id/mainListView"
+            android:layout_width="fill_parent"
+            android:layout_height="fill_parent"
+            android:layout_alignParentRight="true"
+            android:layout_margin="@dimen/main_list_view_margin"
+            android:dividerHeight="@dimen/main_list_view_divider_height"
+            android:divider="@color/abc_input_method_navigation_guard"
+            android:scrollbars="vertical"
+            android:fastScrollEnabled="true"
+            android:fastScrollAlwaysVisible="true"
+            android:background="@android:color/white"
+            android:fadeScrollbars="true"
+            android:backgroundTint="@color/abc_secondary_text_material_dark"
+            android:clickable="true" >
+        </ListView>
```

Comment:
You don't need to close the ListView like this. You can just close it using `/>` like in the other elements